### ISSUE TITLE
Incorrect data parsed from Infobox

### ISSUE THREAD

**Issue by admehta01** [dijs/infobox-parser#33]
https://en.wikipedia.org/w/index.php?title=All_Around_the_World_(Lisa_Stansfield_song)&action=edit

```
{{Infobox song
| name       = All Around the World
| cover      = Lisa Stansfield - All Around the World.jpg
| alt        =
| border     = yes
| caption    = Artwork for releases outside North America
| type       = single
| artist     = [[Lisa Stansfield]]
| album      = [[Affection (Lisa Stansfield album)|Affection]]
|B-side="Wake Up Baby" (7")<br/>"The Way You Want It" (12")| released   = 16 October 1989
| recorded   = 1989
| studio     =
| venue      =
| genre      = 
| length     = {{Duration|m=4|s=28}}
| label      = [[Arista Records|Arista]]
| writer     = {{flat list |
*Lisa Stansfield
*Ian Devaney
*Andy Morris
}}
| producer   = {{flat list |
*Ian Devaney
*Andy Morris}}
| prev_title = [[This Is the Right Time]]
| prev_year  = 1989
| next_title = [[Live Together]]
| next_year  = 1990
| misc       = {{Extra chronology
 | artist     = 2003
 | type       = Singles
 | prev_title = [[8-3-1]]
 | prev_year  = 2001
 | title      = All Around the World (Norty Cotto Mixes)
 | year       = 2003
 | next_title = [[Too Hot (Kool & the Gang song)|Too Hot]]
 | next_year  = 2004
}}
```
returns
```
{
  general: {
    name: 'All Around the World',
    cover: 'Lisa Stansfield - All Around the World.jpg',
    border: true,
    caption: 'Artwork for releases outside North America',
    type: 'Singles',
    artist: '2003',
    album: 'Affection (Lisa Stansfield album)',
    bSide: '"Wake Up Baby" (7"),"The Way You Want It" (12")',
    released: '16 October 1989',
    recorded: '1989',
    length: 'Duration',
    label: 'Arista Records',
    writer: 'flat list',
    producer: 'flat list',
    prevTitle: '8-3-1',
    prevYear: '2001',
    nextTitle: 'Too Hot (Kool & the Gang song)',
    nextYear: '2004',
    misc: 'Extra chronology',
    title: 'All Around the World (Norty Cotto Mixes)',
    year: '2003'
  },
  tables: [],
  lists: []
}
```

Issues:
- artist: '2003' is off
- writer and producer say "flat list"

**Comment by dijs** [comment]
Thanks for the example and source. I will look into this. It seems like it is two different parsing issues.

**Comment by dijs** [comment]
Published to v3.6.2
